pkgs/by-name/rr/rrdtool/package.nix

```
@@ -0,0 +1,69 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  fetchpatch,
+  autoreconfHook,
+  gettext,
+  perl,
+  pkg-config,
+  libxml2,
+  pango,
+  cairo,
+  groff,
+  tcl,
+  darwin,
+}:
+
+perl.pkgs.toPerlModule (
+  stdenv.mkDerivation rec {
+    pname = "rrdtool";
+    version = "1.9.0";
+
+    src = fetchFromGitHub {
+      owner = "oetiker";
+      repo = "rrdtool-1.x";
+      rev = "v${version}";
+      hash = "sha256-CPbSu1mosNlfj2nqiNVH14a5C5njkfvJM8ix3X3aP8E=";
+    };
+
+    # Fix darwin build
+    patches = lib.optional stdenv.isDarwin (fetchpatch {
+      url = "https://github.com/oetiker/rrdtool-1.x/pull/1262.patch";
+      hash = "sha256-aP0rmDlILn6VC8Tg7HpRXbxL9+KD/PRTbXnbQ7HgPEg=";
```

```suggestion
      url = "https://github.com/oetiker/rrdtool-1.x/commit/8a77f3e4123d13b18f712454e038892c4ccf7df2.patch";
      hash = "sha256-OwSa2mBrWes6lHHMsvsv5lNcO3bs2fmEO5P7lf5JNfc=";
```
better use the commit and not the PR as a `url`

---

I don't think we have a policy here. The PR won't change as it has already been merged.

---

Good question. I'm not sure whether we have a best practice for this. As you pointed out, the PR is already merged and shouldn't change. 
I just googled a bit and tried it myself, but a merged PR [seems to be locked](https://stackoverflow.com/questions/12674304/github-reopening-a-merged-pull-request) forever, so this should be fine.